ISSUE #2955 - Dropdown shows correct selected project

Amantini1997 opened the pull request:
This fixes #2955

#### Description
The selected project is correctly highlighted from the dropdown menu

#### Test cases
Select a project which is not the first one and see that it gets correctly highlighted

sanmont3drepo commented:
- I know that this issue specifies only selected project, but , can we see if there's a way to make it work with both the project and the teamspace?

Problem with teamspace:
![image](https://user-images.githubusercontent.com/42032433/150323429-9a0875aa-16e8-4763-acda-c3ed28af8616.png)


- There's a commit  (as well as the PR name) that has the wrong issue number. To fix this you can use https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Amantini1997 commented:
> 



> * I know that this issue specifies only selected project, but , can we see if there's a way to make it work with both the project and the teamspace?
> 
> Problem with teamspace: ![image](https://user-images.githubusercontent.com/42032433/150323429-9a0875aa-16e8-4763-acda-c3ed28af8616.png)
> 
> * There's a commit  (as well as the PR name) that has the wrong issue number. To fix this you can use https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History ( the interactive bit)

I fixed the commits issue number.

I also fixed the selection for all the items of the breadcrumbs, but I adopted a non-dynamic solution. This means that, should the breadcrumbs display more (or different) items other than _teamspace_ and _project_, the `navigationMenu.component` should be modified to account for that.

sanmont3drepo commented on frontend/src/v5/ui/components/shared/navigatonMenu/navigationMenu.component.tsx:
```diff
@@ -42,11 +44,19 @@ export const NavigationMenu = ({ anchorEl, handleClose, list }: INavigationMenu)
 			horizontal: 'left',
 		},
 	};
+	const { project, teamspace } = useParams();
+	const selectedItem = project || teamspace;
```
Can we move this to breadcrumbs.component and pass a selectedItem prop to this component?  
I think we might need to refactor that component (I dont believe is easy to read), but for now I think is better to have this decision there. Which is the component that 'knows' which option has been selected.

Amantini1997 replied:
Done
